src/Utilities/Compiler/DiagnosticCategoryAndIdRanges.txt

```diff
@@ -12,7 +12,7 @@
 Design: CA2210, CA1000-CA1070
 Globalization: CA2101, CA1300-CA1310
 Mobility: CA1600-CA1601
-Performance: HA, CA1800-CA1838
+Performance: HA, CA1800-CA1842
```

Comment:
Why the jump from 38 to 42?

Reply:
Because my other analyzer PRs use CA1839-CA1841.

Reply:
Makes sense, maybe you could add a [X/Y] prefix on your PRs so that we merge them in the right order.

Reply:
I've prefixed them [(A)](https://github.com/dotnet/roslyn-analyzers/pull/4687), [(B)](https://github.com/dotnet/roslyn-analyzers/pull/4726), [(C)](https://github.com/dotnet/roslyn-analyzers/pull/4764), and this one is (D). I think some of them may be in different milestones. Do you want me to change the IDs around so that earlier milestones get merged first?